### PR TITLE
Add scss for components in angular apps by default.

### PR DESCRIPTION
**Pull request by igor-dv**
Issue: #2688

Looks like most of the people (I don't know if it's most, but that what I've seen recently) prefer using scss instead of css. 

## What I did
Changed the raw-loader to use scss

For components that use `styleUrls` - scss will be loaded with a raw-loader
For global imports - there will be still style-loader as before.

### CODE REVIEW

**Review by amcdnl — APPROVED** [review]
LGTM

**Comment by Hypnosphi** [comment]
Is it a breaking change?

**Comment by amcdnl** [comment]
@Hypnosphi - I believe its just for `styleUrls` if I'm correct.

File: app/angular/src/server/config/webpack.config.js [hunk 1]
```diff
@@ -67,9 +67,13 @@ export default function(configDir) {
           loaders: [require.resolve('ts-loader'), require.resolve('angular2-template-loader')],
         },
         {
-          test: /\.(html|css)$/,
+          test: /\.html$/,
           loader: 'raw-loader',
-          exclude: /\.async\.(html|css)$/,
+          exclude: /\.async\.css$/,
```
Comment (Hypnosphi) [inline app/angular/src/server/config/webpack.config.js]: Why do we need to exclude a css file is it doesn't pass the `test` anymore?

**Comment by Hypnosphi** [comment]
> its just for `styleUrls` if I'm correct

This PR changes `styleUrls` in example app. Does it mean that users will need to change it as well?

File: app/angular/src/server/config/webpack.config.js (hunk 1 above)
Comment (igor-dv) [reply app/angular/src/server/config/webpack.config.js]: oops, I think I need to exclude html here.

**Comment by igor-dv** [comment]
@Hypnosphi, it feels like a breaking change, but I prefer to look at it as a bug fix. Before, in order to make css work, people needed to extend webpack.config + exclude thigngs from css rules (because css didn't really work by default) + add scss. So now they maybe will need just to remove these extended webpack.config ...

**Review by ndelangen — APPROVED** [review]

**Comment by Hypnosphi** [comment]
> css didn't really work by default

OK then

**Pull request merged**